exec/java-exec/src/main/resources/rest/static/js/ace-code-editor/snippets/sql.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Drill SQL Syntax Snippets
+ */
+
+ace.define("ace/snippets/sql",["require","exports","module"], function(require, exports, module) {
+"use strict";
+
+exports.snippetText = "snippet info\n\
+	select * from INFORMATION_SCHEMA.${1:<tableName>};\n\
+snippet sysmem\n\
```

Comment:
Also snippets are case-sensitive. Is it common practice?

Reply:
Not sure what you mean.
All available snippet keywords (e.g. 'info') are in lower-case, but I don't think that is affecting the suggested autocomplete options. The template, however, is projected verbatim to the definition. So, INFORMATION_SCHEMA will remain in upper case. 
![image](https://user-images.githubusercontent.com/4335237/34736399-a2c01e78-f527-11e7-84aa-fcdfcefc6a5c.png)

Reply:
Turns out that my Chrome, it works when using -> Ctrl + space.
Could you please update Jira with list of available snippets and ways how to enable them?